unit_test/async_batcher_test.cc

```diff
@@ -84,12 +112,26 @@ namespace reinforcement_learning { namespace logger {
 
     static size_t size_estimate(const test_undroppable_event& evt) { return 1; }
   };
+    
+  template <>
+  struct json_event_serializer<config_drop_event> {
+    using serializer_t = json_event_serializer<config_drop_event>;
+
+    static int serialize(config_drop_event& evt, std::ostream& out, api_status* status) {
+      out << evt.get_event_id();
+      return error_code::success;
+    }
+
+    static size_t size_estimate(const config_drop_event& evt) { return 1; }
```

Comment:
Isn't the size the length of the event id?

Reply:
I'm not sure, I just copied it from the previous examples. It probably is? But I assume this estimate is only used to try and optimize the number of memory allocations, I dont think anything will break from it being wrong

Reply:
Size estimate can be anything so any positive value should work.